NEWS.md

```diff
@@ -1,5 +1,12 @@
 # forcats (development version)
 
+* `fct_reorder()` now removes `NA` values in `.x` with a warning (like
+   `ggplot2::geom_point()` and friends). You can suppress the warning by
+   setting `.na_rm = TRUE` (#315).
+   
+* `fct_reorder()` gains a new `.na_last` argument that allows you to control
+  where levels that are empty or only contain missing values are placed (#266).
```

Comment:
This description doesn't seem quite right. I don't think the `NA`s appear in the _levels_, they appear in `.x`.

If you keep the current `.na_rm` behavior, is this really ever used?

Reply:
I now see where this is used down in the tests. See more comments there.

But I still think the description isn't quite right. It's like it is used for:
- Levels that don't appear in the data (and `.default = NA`)
- When `.x` has missing values (and `.na_rm = FALSE`)